app/lib/package/models.dart

```diff
@@ -66,9 +66,15 @@ class Package extends db.ExpandoModel<String> {
   @db.ModelKeyProperty(propertyName: 'latest_version', required: true)
   db.Key latestVersionKey;
 
+  @db.DateTimeProperty()
+  DateTime latestPublished;
+
   @db.ModelKeyProperty(propertyName: 'latest_dev_version')
   db.Key latestPrereleaseVersionKey;
```

Comment:
```suggestion
  /// Reference to latest version of this package ordered by semantic versioning, including
  /// prerelease versions of this package.
  ///
  /// Note. This is **not** the _latest prerelease version_, it is the _latest version_ without deprioritization of
  /// prereleases. Hence, this might not be a prerelease version, if a newer non-prerelease exists.
  @db.ModelKeyProperty(propertyName: 'latest_dev_version')
  db.Key latestPrereleaseVersionKey;
```
Is this correct? I think this is worth documenting, because the property is NOT what it says :D

Also can this be `null`? We should document this.

Reply:
The update is correct, thanks! Maybe we could migrate away from the old notion of storing the key on the entity, and with that also rename it?

I think it cannot be `null`, and we could probably mark it as such.